dev/TeachingTip/InteractionTests/TeachingTipTestPageElements.cs

```diff
@@ -358,6 +358,30 @@ public TextBlock GetEffectiveForegroundOfTeachingTipContentTextBlock()
         }
         private TextBlock effectiveForegroundOfTeachingTipContentTextBlock;
 
+        public Button GetRemoveTeachingTipButton()
+        {
+            return GetElement(ref effectiveRemoveTeachingTipButton, "RemoveTeachingTipButton");
+        }
+        private Button effectiveRemoveTeachingTipButton;
+
+        public TextBlock GetTeachingTipContent()
+        {
+            return GetElement(ref effectiveTeachingTipContent, "TeachingTipContentTextBlock");
+        }
+        private TextBlock effectiveTeachingTipContent;
+
+        public CheckBox GetToolTipContentUnloadedCheckbox()
```

Comment:
>GetToolTipContentUnloadedCheckbox [](start = 24, length = 33)

Only just noticed, this should be TeachingTip not ToolTip

Reply:
I changed that since we are already changing the file in this PR.